ci/bamboo/build-manylinux-prototype.sh

```diff
@@ -1,101 +0,0 @@
-#!/bin/bash
```

Comment:
The generic portion could then probably be used for OS X builds, too.

I realize that this excludes Windows. Windows has toolchain installation requirements that differ from Linux and OS X, as well as additional parameters for cmake, and probably other specifics. We might evolve the "builder" into a 3-platform shared script written in Python, depending on how much info needs to be passed around versus the DRY savings.